sample_information_validator.py

```diff
@@ -51,44 +60,97 @@ def validate(self, cell):
             return False
         except TypeError:
             if cell.value is None:
-                logger.warning(
+                logger.error(
                     'Cell {} is numeric but empty'.format(cell.coordinate)
                 )
+                return False
             else:
                 raise
 
+    def validate_vol(self,cell): # TODO different volumes for different input types! requires changes in data sheet
+        """Checks entry for volume"""
+        if cell.value is None:
+            logger.error(
+                'No sample volume given in cell {}'.format(cell.coordinate)
+            )
+            return False
+        else:
+            return True
+
+    def validate_rin(self,cell, sample_type):
+        """Checks entry for RIN in RNA samples only"""
+        if cell.value is None:
+            logger.error(
+                'No RIN value given in cell {}'.format(cell.coordinate)
+            )
+            return False
+        elif cell.value <8:
+            logger.warning(
+                'RIN value in cell {} is below recommendation'.format(cell.coordinate)
+            )
+            return True
+        else:
+            return True
 
-def validate_column(sheet, column_letter, row_start,
-                    row_end, validator, validator_attr={}):
-    """Validates a section of a column
+def validate_column(sheet, plate_id, concentration_letter, volume_letter, rin_letter, cell_sample_type, sample_rowID,
```

Comment:
In its current form, I think the contents of this function could be moved to 'main'.

Reply:
Ok, that's fine. But then the name validate_column is a bit strange, no? Could be just validate, or what do you imagine this specific function to do?